common.tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "pretty": true,
     "strictNullChecks": true,
     "noUnusedLocals": true,
+    "esModuleInterop": true,
```

Comment:
What prompted the need for this?

Reply:
Was seeing this: 

```
import MongoNav
Cannot use JSX unless the '--jsx' flag is provided.ts(17004)
```

All across the repo